src/diamond/handler/pubsub.py

```diff
@@ -0,0 +1,216 @@
+# coding=utf-8
+
+"""
+Send metrics to [Google Pub/Sub](https://cloud.google.com/pubsub/).
+
+#### Copyright
+
+Copyright 2016 Google Inc.
```

Comment:
FWIW, I'm a-okay with this. Having a copyright listed only really will pertain if we ever re-license away from the MIT license we're currently under. If we re-license, we would have to seek approval from every contributor anyway.